app/bingey-ui/src/components/Watchlist/TitleCard/TitleCard.jsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState, useEffect } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+import CardActionArea from '@material-ui/core/CardActionArea';
+import CardActions from '@material-ui/core/CardActions';
+import CardContent from '@material-ui/core/CardContent';
+import CardMedia from '@material-ui/core/CardMedia';
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
+import './cardStyle.scss';
+//import TitleDialog from './TitleDialog';
+
+const useStyles = makeStyles({
+  root: {
+    maxWidth: 345,
```

Comment:
I don't think hard-coding pixel values is good (same for the `height`). Using `aspect-ratio` seems to work nice. Also, the `Grid` component handles the width of the items inside it.